src/Adapter/Driver/IbmDb2/Result.php

```diff
@@ -176,10 +177,11 @@ public function getFieldCount()
     }
 
     /**
-     * @return null|int
+     * @return int
      */
+    #[ReturnTypeWillChange]
     public function count()
     {
-        return;
+        return 0;
```

Comment:
Whoa, what is this funky business? I know DB2 is a steaming pile of... "legacy", but this seems borked :D

Reply:
I'm a bit shocked we were not returning AN INTEGER from a `count()` method as well!